src/lib/openapi/spec/client-app-schema.ts

```diff
@@ -0,0 +1,63 @@
+import { FromSchema } from 'json-schema-to-ts';
+import { parametersSchema } from './parameters-schema';
+
+export const clientAppSchema = {
```

Comment:
Could call this `clientMetricsSchema` since that's the name of its controller.

Would be nice to separate it from `clientApplicationSchema` at least.